wiki/Contests/TKBC/2/en.md

```diff
@@ -0,0 +1,77 @@
+---
+tags:
+- TKBC
+- TKBC2
+---
+
+# Taiko Korean Beatmap Contest #2
+
+![TKBC2 Logo](img/logo.png)
+
+The **Taiko Korean Beatmap Contest #2** (***TKBC2***) was a osu!taiko mapping contest for Koreans. It was the second installment of the TKBC.
+
+## Contest schedule
+
+| Event | Timestamp (UTC +9) |
+| --: | :-- |
+| Submitting Phase | 2020-11-30/2020-12-27 |
+| Judging Phase | 2020-12-28/2021-01-17 |
+| Results Announcement | 2021-01-17 |
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
+- The difficulty should be Oni or higher.
+- You must follow the [Ranking Criteria](/wiki/Ranking_Criteria).
+- If you submit several maps, the last one would be your entry.
+- You should not reveal your map before the results announcing.
+- You can't submit what you uploaded already.
+- Only Koreans can participate this contest.
+- Judges can't participate this contest.
+
+### Metadata rules
+
+- Artist must be `asdf`.
+- Title must be `1223`.
+- Difficulty must be Oni or Inner Oni.
+- Source and tags must be left blank.
+- When submitting your map, include the metadata in the email.
+
+## Theme
+
+- Snow
+- Star
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile Badge |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of supporter |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of supporter |
+
+## Organisation
+
+The Taiko Korean Beatmap Contest #2 is run by members of the Korean osu!taiko mapping community.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | [[Karcher] 2](https://osu.ppy.sh/users/9892196), [Civil oath](https://osu.ppy.sh/users/3216107) |
```

Comment:
a lot of [escape characters](http://bookdown.org/yihui/rmarkdown-cookbook/special-chars.html) are seeming to be missing on these tables at the moment, namely :

- `[[Karcher] 2]` should be --> `[\[Karcher\] 2]`
- `[4_4]` should be --> `[4\_4]`